cfme/tests/cloud_infra_common/test_vm_ownership.py

```diff
@@ -8,16 +8,30 @@
 from cfme.common.vm import VM
 from cfme.exceptions import VmOrInstanceNotFound
 from cfme.utils.blockers import BZ
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
 
 
 pytestmark = [
     test_requirements.ownership,
     pytest.mark.meta(blockers=[BZ(1380781, forced_streams=["5.7"])]),
     pytest.mark.tier(3),
-    pytest.mark.provider([BaseProvider], required_fields=['ownership_vm'], scope='module')
+    pytest.mark.provider([BaseProvider], scope='module')
```

Comment:
**Required**

Restrict this to CloudInfraProvider, I don't think you'll want to generate against network providers, physical infra providers, or container providers - yet.